app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
Foreign key is improperly implemented. As per the given ERD, the Comment association should have Comment belong to User through Author, an alias of User, like below:
`belongs_to :author, class_name: 'User'`

Kindly ensure to improve upon this aspect.

Reply:
Hello @petudeveloper I do not think I need to use class_name: 'User' as the user_id is the foreign key I am using and that will automatically match the User class

Reply:
@Whoistolu  Actually the Code Reviewer is correct. You should have` belongs_to :author, class_name: 'User'`. Let's go back to your [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

Reply:
@arslanbisharat I have implemented the recommended changes, although I was using `User` which has a `user_id` in the `schema` so rails will automatically figure that out.